framework/test/mocha/functional/http/get/transactions.js

```diff
@@ -146,11 +151,15 @@ describe('GET /api/transactions', () => {
 			return waitFor
 				.confirmations(_.map(transactionList, 'id'))
 				.then(() => {
-					return apiHelpers.sendTransactionPromise(transaction3);
+					const promises2 = [];
+					promises2.push(apiHelpers.sendTransactionPromise(transaction3));
+					promises2.push(apiHelpers.sendTransactionPromise(transactionType3));
+					return Promise.all(promises2);
```

Comment:
I think having promises2 array doesn't bring that much value. Maybe it's better to move the calls inside Promise.all as an array ?

Reply:
Yes, you are right, looks weird now, thanks for fast review!